components/feature/searchwidget/src/main/java/mozilla/components/feature/searchwidget/IntentUtils.kt

```diff
@@ -0,0 +1,23 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.feature.searchwidget
+
+import android.app.PendingIntent
+import android.os.Build
+
+object IntentUtils {
```

Comment:
This is already available in `support.utils` which can be added as a dependency to the new feature.

Reply:
Done